File: winform_client/winform_client/entity/CustomAppMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace winform_client.entity
{
    class CustomAppMessage
    {
        public string command { get; set; }
        public string sendToImei { get; set; }
        public Object content { get; set; }
    }
}

File: android_client/app/src/main/java/com/github/duc010298/android/entity/socket/CustomAppMessage.java
package com.github.duc010298.android.entity.socket;

public class CustomAppMessage {
    private String command;
    private String sendToImei;
    private Object content;

    public String getCommand() {
        return command;
    }

    public void setCommand(String command) {
        this.command = command;
    }

    public String getSendToImei() {
        return sendToImei;
    }

    public void setSendToImei(String sendToImei) {
        this.sendToImei = sendToImei;
    }

    public Object getContent() {
        return content;
    }

    public void setContent(Object object) {
        this.content = content;
    }
}
